docs/redirects.md

```diff
@@ -0,0 +1,73 @@
+# Adding redirects in MDN
+
+MDN has a series of redirects, defined in the file
+[\_redirects.txt](https://github.com/mdn/content/blob/main/files/en-us/_redirects.txt) in the `content` repo.
+
+Each line contains a single redirect, in the form:
+
+```bash
+FROM-URL  TO-URL
+```
+
+The relative URL to redirect from, followed by a tab character, followed by the
+relative URL to redirect to. A real example is as follows:
+
+```bash
+/en-US/docs/AJAX  /en-US/docs/Web/Guide/AJAX
+```
+
+One built, the Yari system then puts all of these redirects in place.
+
+The recommended way to add redirects correctly while avoiding errors is to use
+the `add-redirect` tool available in the [yari](https://github.com/mdn/yari)
+repo. Let's see how to use this.
+
+## Preparation
+
+- First of all fork and locally clone the `yari` and `content` repos. If you
+  already have some of them cloned locally, make sure they are up-to-date
+  (e.g. `fetch` the updated content from the origin repos).
+- Install the latest toolset in the `yari` repo using `yarn install`.
+- Create a new branch in the `content` repo, and check
+  it out locally.
+- `cd` into your `yari` repo.
```

Comment:
The `yarn tool` command is also available in the content repo.

_Just_ clone the content repo and cd inside then run `yarn` once. Afterwards one can run `yarn content` from within the content repo `yarn content --help` so see the options. That the same as running `yarn tool` from the yari repo.